src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costs = 40;
+
+  if (days < 3) {
+    return days * costs;
+  } else if (days >= 3 && days < 7) {
+    return (days * costs) - 20;
+  } else {
+    return (days * costs) - 50;
+  }
```

Comment:
```suggestion
  }
  
  return (days * costs) - 50;
```
тут else омжно убрать